pkg/chunk/cache/redis_client.go

```diff
@@ -16,16 +16,17 @@ import (
 
 // RedisConfig defines how a RedisCache should be constructed.
 type RedisConfig struct {
-	Endpoint    string         `yaml:"endpoint"`
-	MasterName  string         `yaml:"master_name"`
-	Timeout     time.Duration  `yaml:"timeout"`
-	Expiration  time.Duration  `yaml:"expiration"`
-	DB          int            `yaml:"db"`
-	PoolSize    int            `yaml:"pool_size"`
-	Password    flagext.Secret `yaml:"password"`
-	EnableTLS   bool           `yaml:"enable_tls"`
-	IdleTimeout time.Duration  `yaml:"idle_timeout"`
-	MaxConnAge  time.Duration  `yaml:"max_connection_age"`
+	Endpoint      string         `yaml:"endpoint"`
+	MasterName    string         `yaml:"master_name"`
+	Timeout       time.Duration  `yaml:"timeout"`
+	Expiration    time.Duration  `yaml:"expiration"`
+	DB            int            `yaml:"db"`
+	PoolSize      int            `yaml:"pool_size"`
+	Password      flagext.Secret `yaml:"password"`
+	TLSEnabled    bool           `yaml:"tls_enabled"`
+	TLSSkipVerify bool           `yaml:"tls_skip_verify"`
```

Comment:
For consistency with other similar config, I would name it `tls_insecure_skip_verify`.